src/lib/schedule/ScheduledTask.js

```diff
@@ -233,7 +233,8 @@ class ScheduledTask {
 	 * @private
 	 */
 	static _generateID(client) {
-		return Date.now().toString(36) + (client.shard ? client.shard.id.toString(36) : '') + String.fromCharCode((1 % 26) + 97);
+		i = i < 46656 ? i + 1 : 0;
```

Comment:
I don't understand why this is needed. You cannot have 2+ at the same time, except for in the case of shards, at which point we have already handled...

Reply:
With internal sharding, the chances to generate two ids at the same time become greatly bigger, but it's also for the case when two shards generate the same id, which can happen in large scale bots.

Reply:
It's impossible with internal sharding because node is single threaded. Unless internal sharding was done with workers, which it isn't currently.

It's impossible for two shards to generate the same id, because the shard id is taken into account.